### PR TITLE
Refactor/account e2e tests

### PR DESCRIPTION
**Pull request by Thistle-Strum**
🔥 Summary
Account page e2e tests
https://github.com/casper-network/casper-blockexplorer-frontend/assets/88854201/0e40da88-4001-4d7d-9364-cc06ef4b839f

### CODE REVIEW

File: test/cypress/e2e/account.cy.ts [hunk 1]
```diff
@@ -1,21 +1,89 @@
...
+
+        cy.getByData('balance-h3').should('contain', 'Balance');
+
+        cy.getByData('raw-data-h3').should('contain', 'Raw Data');
+        cy.getByData('raw-data-button')
+          .should('contain', 'Show Raw Data')
+          .click()
+          .getByData('raw-data-button')
+          .should('contain', 'Hide Raw Data')
+          .find('root":{...}');
```
Comment (Thistle-Strum) [inline test/cypress/e2e/account.cy.ts:60]: [](url)

File: test/cypress/e2e/data/account-test-data.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// from mainnet
+const accountHash: string =
```
Comment (mgibson-casper) [inline test/cypress/e2e/data/account-test-data.ts]: TS is able to infer types, so we don't need to explicitly type here (or line 8 below)

Comment (mgibson-casper) [reply test/cypress/e2e/data/account-test-data.ts]: Please update all instances in this PR as I notice there are a few others.

Comment (Thistle-Strum) [reply test/cypress/e2e/data/account-test-data.ts]: Updated!

File: test/cypress/e2e/home.cy.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ describe('Home Page', () => {
 
       cy.visit('/');
 
-      const homePageHeroText = 'Discover the Casper Blockchain.';
+      const homePageHeroText = 'Discover the Jenkins blockchain.';
```
Comment (DorianKwan) [inline test/cypress/e2e/home.cy.ts:10]: Jenkins eh?

File: test/cypress/e2e/data/account-test-data.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// from mainnet
+const accountHash =
+  'b7e3617083ec93d15ca8d6a570bc9b9b8508b3a877c9cf94fb3d327a69f33b7a';
+const truncatedAccountHash = `${accountHash.slice(0, 5)}...${accountHash.slice(
+  -5,
+)}`;
+
+const publicKey =
+  '018f84c6fc037284f189cc8cb49f89212ff434a5eb050e48cdd164ff3890fbff69';
+const truncatedPublicKey = `${publicKey.slice(0, 5)}...${publicKey.slice(-5)}`;
```
Comment (DorianKwan) [inline test/cypress/e2e/data/account-test-data.ts]: I see a bit of philosophy conflicting here with both of our approaches.  Thoughts on consolidating to one pattern?

**Review by DorianKwan — CHANGES_REQUESTED** [review]

File: test/cypress/e2e/data/account-test-data.ts (hunk 4 above)
Comment (Thistle-Strum) [reply test/cypress/e2e/data/account-test-data.ts]: I'm happy to use the pattern you set up! But it was causing my search-form tests to fail.  Any idea why that is?

File: test/cypress/e2e/home.cy.ts (hunk 3 above)
Comment (Thistle-Strum) [reply test/cypress/e2e/home.cy.ts:10]: Yeah, I'm not sure who Jenkins is. The name change occurred when we first started making the app configurable.

Comment (valfaro123) [reply test/cypress/e2e/home.cy.ts:10]: Check your .env file. This is set under ORG_NAME

File: test/cypress/e2e/data/account-test-data.ts (hunk 4 above)
Comment (Thistle-Strum) [reply test/cypress/e2e/data/account-test-data.ts]: Ah, I think I know why.  Will update!

**Review by mgibson-casper — APPROVED** [review]

File: test/cypress/e2e/home.cy.ts (hunk 3 above)
Comment (DorianKwan) [reply test/cypress/e2e/home.cy.ts:10]: Ahhh ^ that'll do it!

File: test/cypress/e2e/data/index.ts [hunk 5]
```diff
@@ -1 +1,3 @@
 export * from './hashes';
+export * from './account-test-data';
+export * from './search-form-data';
```
Comment (DorianKwan) [inline test/cypress/e2e/data/index.ts]: These files have been deleted in your last commit!  We will need to remove the references to them :)

**Review by DorianKwan — CHANGES_REQUESTED** [review]

**Review by DorianKwan — APPROVED** [review]

**Pull request merged**